docs/guides/features/icons.md

```diff
@@ -1,28 +1,45 @@
+import Command from '@theme/Command'
+
 # Icons
 
-Tauri ships with a default iconset based on its logo. This is probably NOT what you want when you ship your application. To remedy this common situation, Tauri provides the `tauricon` command that will take an input file ("./app-icon.png" by default) and create all the icons needed for the various platforms:
+Tauri ships with a default iconset based on its logo. This is NOT what you want when you ship your application. To remedy this common situation, Tauri provides the `icon` command that will take an input file (`"./app-icon.png"` by default) and create all the icons needed for the various platforms:
 
 ## Usage
 
+Starting with `@tauri-apps/cli` / `tauri-cli` version 1.1 the `icon` is part of the main cli:
```

Comment:
```suggestion
Starting with `@tauri-apps/cli` / `tauri-cli` version 1.1 the `icon` subcommand is part of the main cli:
```

After we update the CLI docs post-launch we can add a link to it as well

Reply:
good call, we need to update the cli doc template before the release x)

Reply:
I'll do a change to test it